src/test/scala/it/pagopa/interop/authorizationprocess/ClientOperationSpec.scala

```diff
@@ -176,10 +176,12 @@ class ClientOperationSpec extends AnyWordSpecLike with MockFactory with SpecUtil
       val consumerUuid: Option[UUID]     = Some(client.consumerId)
       val purposeUuid: Option[UUID]      = Some(clientPurpose.states.purpose.purposeId)
 
+      mockGetTenant()
+
       (mockPartyManagementService
-        .getRelationships(_: UUID, _: UUID, _: Seq[String])(_: Seq[(String, String)], _: ExecutionContext))
+        .getRelationships(_: String, _: UUID, _: Seq[String])(_: Seq[(String, String)], _: ExecutionContext))
         .expects(
-          consumerId,
+          consumerId.toString(),
```

Comment:
this should fail badly, because the aim of this PR is to change this behavior 😅 
(same for allo other party mock calls)

The `mockGetTenant` must assign to `id` and `selfcareId` different values

Reply:
hmm, yes, the thing is that the two ids in the tests are the same, that's why they work. I'll refactor the tests a to make them a bit more meaningful.